deep_qa/layers/tuple_matchers/encoded_tuple_matcher.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, text_trainer, tuple_matcher_params: Dict[str, Any]):
         if tuple_matcher_params is None:
             tuple_matcher_params = {}
         tuple_matcher_choice = get_choice_with_default(tuple_matcher_params,
-                                                       "type",
+                                                       "encoded_match_type",
```

Comment:
Oh, nevermind.  I thought there was some nested structure here.  This is probably fine.  ("probably" because I was wondering if we should make this structure nested...  But that can wait for another PR, if we decide we want to change it.)